disabled_apps/clock/manifest.webmanifest

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Clock",
+  "description": "Gaia Clock",
+  "lang": "en",
+  "short_name": "Clock",
+  "scope": "./",
+  "start_url": "./index.html?homescreen=1",
+  "launch_path": "/index.html",
```

Comment:
We shouldn't need both start_url and launch_path. Is start_url not working for something?